components/Controls.tsx

```diff
@@ -70,8 +70,21 @@ function BikeStats({ bike }: { bike: Bike }) {
     )
 }
 
+function isNewerFirmwareVersion (newVer: string, oldVer: string) {
+    const oldParts = oldVer.split('.')
+    const newParts = newVer.split('.')
+    for (var i = 0; i < newParts.length; i++) {
+        const a = ~~newParts[i] // parse int
+        const b = ~~oldParts[i] // parse int
```

Comment:
I find `~~` a wired way to convert a string to a number, please use `Number(..)` instead
```suggestion
        const a = Number(newParts[i])
        const b = Number(oldParts[I])
```

Reply:
I would like to accept your suggestion but at line 78 you capitalised the letter 'i' so I will commit this manually with out the capitalised letter.